src/worker/tasks/processTx.ts

```diff
@@ -171,11 +171,27 @@ export const processTx = async () => {
                 ...gasOverrides,
               });
 
-              // TODO: We need to target specific cases
-              // Bump gas limit to avoid occasional out of gas errors
-              txRequest.gasLimit = txRequest.gasLimit
-                ? BigNumber.from(txRequest.gasLimit).mul(120).div(100)
-                : undefined;
+              // Gas limit override
+              if (tx.gasLimit) {
+                txRequest.gasLimit = BigNumber.from(tx.gasLimit);
+              } else {
+                // TODO: We need to target specific cases
+                // Bump gas limit to avoid occasional out of gas errors
+                txRequest.gasLimit = txRequest.gasLimit
+                  ? BigNumber.from(txRequest.gasLimit).mul(120).div(100)
+                  : undefined;
+              }
```

Comment:
```suggestion
} else if (txRequest.gasLimit) {
  // Use +20% gas limit to prevent occasional gas errors.
  txRequest.gasLimit = BigNumber.from(txRequest.gasLimit).mul(120).div(100)
}
```

Reply:
If they pass a gas limit we should probably not modify it, default without gas limit overrides already adds overheads

Reply:
I believe this was added for shield (ran into issues running out of gas on airdrops). We could remove it and see if it returns?